common/ssl_calls.c

```diff
@@ -592,6 +647,15 @@ ssl_tls_accept(struct ssl_tls *self, long ssl_protocols,
                      SSL_MODE_ACCEPT_MOVING_WRITE_BUFFER |
                      SSL_MODE_ENABLE_PARTIAL_WRITE);
     SSL_CTX_set_options(self->ctx, options);
+    DH *dh = get_dh2236 ();
```

Comment:
Check return value is not `NULL`?

Reply:
Actually yes we should check it's not ```NULL```, it could happen if ```DH_new()``` returns ```NULL```